data-transforms/flatten/transform.yaml

```diff
@@ -0,0 +1,4 @@
+name: flatten
+input-topic: ""
+output-topic: ""
+language: tinygo
```

Comment:
if you use `tinygo-with-goroutines` then we'll automatically add the asyncify bit: 

https://github.com/redpanda-data/redpanda/pull/15103

Reply:
Thanks. Updated that.